setup.py

```diff
@@ -2,8 +2,10 @@
 
 setup(
     name="rich-click",
+    entry_points={"console_scripts": ["rich-click = rich_click.cli:main"]},
```

Comment:
Not sure how to write this in setup.cfg, so here it is. Let me know if you'd still prefer it there.

Reply:
Looks like it can be done: https://stackoverflow.com/a/48891252/713980

```
[options.entry_points]
console_scripts =
    rich-click = rich_click.cli:main
```